Sources/SnapshotTesting/Common/View.swift

```diff
@@ -852,7 +852,11 @@ extension View {
                 callback(Image())
                 return
               }
-              wkWebView.takeSnapshot(with: nil) { image, _ in
+              let configuaration = WKSnapshotConfiguration()
+              if #available(iOS 13, macOS 10.15, *) {
+                configuaration.afterScreenUpdates = false
+              }
+              wkWebView.takeSnapshot(with: configuaration) { image, _ in
```

Comment:
Spelling error configuaration -> configuration

```suggestion
              let configuration = WKSnapshotConfiguration()
              if #available(iOS 13, macOS 10.15, *) {
                configuration.afterScreenUpdates = false
              }
              wkWebView.takeSnapshot(with: configuration) { image, _ in
```